### PR TITLE
Create directories & subdirectories for photo files

### PR DESCRIPTION
**Pull request by kimlew**
Creates year & month sub-directories - to accompany the previous code to change filenames based on photo-taken date

@kcharter Can you please review the recent changes? Thanks.

### CODE REVIEW

File: change_date_rename_file.sh [hunk 1]
```diff
@@ -80,7 +80,22 @@ while read a_file_name; do
...
+  year=${exif_date:0:4}
+  month=${exif_date:5:2}
+
+  just_path=$(dirname "${new_file_name}")
+  path_with_subdir_year_month="${just_path}/${year}/${month}"
+
+  mkdir -p ${path_with_subdir_year_month}
+
+  just_filename=$(basename "${new_file_name}")
+  new_dir_and_filename=$just_path/$year/$month/$just_filename
```
Comment (kcharter) [inline change_date_rename_file.sh]: In your other parameter expansions, like `${just_path}/${year}/${month}` above, you're using curly braces around the variable names, but here you don't. It's not wrong, but you should use a consistent style and add the curly braces here.

**Review by kcharter — COMMENTED** [review]
This looks good. I've got just one very simple suggestion.

File: change_date_rename_file.sh (hunk 1 above)
Comment (kimlew) [reply change_date_rename_file.sh]: @kcharter As you suggested, I made the change so it is more consistent with other brace expansions in script.

**Review by kcharter — APPROVED** [review]
Looks good!

**Pull request merged**